openpathsampling/ensemble.py

```diff
@@ -2271,7 +2297,8 @@ def __call__(self, trajectory, trusted=None, candidate=False):
         trajectory : :class:`openpathsampling.trajectory.Trajectory`
             The trajectory to be checked
         """
-        for frame in trajectory.as_proxies():
+        itraj = getattr(trajectory, 'iter_proxies', trajectory.__iter__)
+        for frame in list(itraj()):
```

Comment:
Why `list` here?

Reply:
(Note: if you do a separate `_get_list_traj` function, feel to use it here and make the list -- I'm not worried about that extra overhead. I just wondered why it was explicitly made a list on this particular line, since it doesn't seem to need to be.)

Reply:
> Why list here?

No reason, just me trying to code in the early morning :laughing: . You are absolutely right.

About the 
> - I'm not worried about that extra overhead.

It would not actually be extra overhead compared to `master` as that is exactly what `as_proxies()` does (it returns a list wrapped iterator). And for the ease of implementation I will actually implement it that way.

Reply:
> It would not actually be extra overhead compared to `master`

Sure -- what I really meant was "Don't feel like you need to create two functions; one that does the iterator and one that does the list. Unless a trajectory has a million frames (which it shouldn't) you might as well just do the list. And don't think that my question about using a list here means you need to use an iterator everywhere possible."